determined-ai/determined
# feat: pin experiments

## Description
[DET-8135]





## Test Plan
- Check if experiment item can be pinned or unpinned
- Check if experiment items stick to the top of table
- Check if there are pinned items on each pagination page on top
- Check if pagination works well
- Check if pinned experiment should be removed after Move Action
- Check if pin experiments up to 5 in each project 




## Commentary (optional)
- Netlify doesn't work as expected cuz i changed backend 




## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-8135]: https://determinedai.atlassian.net/browse/DET-8135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
